packages/ui/Button/index.tsx

```diff
@@ -9,6 +9,7 @@ interface Button extends React.ButtonHTMLAttributes<HTMLButtonElement> {
   size?: 'small' | 'medium' | 'large';
   url?: string;
   noAnimation?: boolean;
+  bordeRadius?: string;
```

Comment:
remove this property please, just use the className

Reply:
removed